### PR TITLE
FIX: PEP8 in test_imaffine

### PR DESCRIPTION
**Pull request by riddhishb**
pep 8 for test_imaffine.py. Issue #864

### CODE REVIEW

File: dipy/align/tests/test_imaffine.py [hunk 1]
```diff
@@ -467,7 +482,8 @@ def test_affine_map():
             expected_nn = oracle_nn(img, dom_shape[:dim], aff_inv_inv)
 
             affine_map = imaffine.AffineMap(aff_inv,
-                                            cod_shape[:dim], codomain_grid2world,
+                                            cod_shape[
+                                                :dim], codomain_grid2world,
```
Comment (arokem) [inline dipy/align/tests/test_imaffine.py]: I prefer not to break in the middle of an indexing operation. Can you break this line after the comma?

**Comment by arokem** [comment]
One small comment. Other than that, looks good.

**Comment by riddhishb** [comment]
Check Now

**Comment by arokem** [comment]
Yeah. This now looks good to me. If someone else wants to take a look, that would be great.

For some reason, the Travis CI system was turned off for dipy. I just flipped the switch back on for that, so future commits on all PRs should trigger a run of the tests.

**Comment by arokem** [comment]
Tests pass on my laptop. I'm merging.

**Pull request merged**